app/containers/MessageActions/index.js

```diff
@@ -90,7 +91,7 @@ const MessageActions = React.memo(forwardRef(({
 			return false;
 		}
 		const deleteOwn = isOwn(message);
-		if (!(permissions.hasDeletePermission || (Message_AllowDeleting && deleteOwn) || permissions.hasForceDeletePermission)) {
+		if (!(permissions.hasOwnDeletePermission || (Message_AllowDeleting && deleteOwn) || permissions.hasForceDeletePermission)) {
```

Comment:
you must still check `hasDeletePermission`, just change our `const = deleteOwn` to check the permission instead of just allow the owner to delete.